src/metagen/src/fdk_python/static/types.py.jinja

```diff
@@ -96,7 +98,8 @@ def __repr(value: Any):
 def typed_{{ func.name }}(user_fn: Callable[[{{ func.input_name }}], {{ func.output_name }}]):
     def exported_wrapper(raw_inp):
         inp: {{ func.input_name }} = Struct.new({{ func.input_name }}, raw_inp)
-        out: {{ func.output_name }} = user_fn(inp)
+        ctx: {{ func.input_name}} = Ctx()
```

Comment:
I think you should skip the type hint here, because it should be just `Ctx`

Reply:
Okay